test/disputes/query.test.ts

```diff
@@ -516,7 +517,7 @@ describe('DisputeManager:Query', async () => {
       await disputeManager.connect(arbitrator.signer).acceptDispute(dID1)
       // Check
       const relatedDispute = await disputeManager.disputes(dID2)
-      expect(relatedDispute.indexer).eq(AddressZero)
+      expect(relatedDispute.status).not.eq(3) // 3 = DisputeStatus.Pending
```

Comment:
It'd be better to explicitly check for the status we expect here, rather than the one we don't. It'd also be nice to add similar checks in other tests here to have better coverage and ensure the status changes work as expected for other cases.